utils/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -52,6 +52,18 @@ class HttpFileStorage(
      */
     private val cacheMaxAgeInSeconds: Int = 0
 ) : FileStorage {
+    override fun exists(path: String): Boolean {
+        val request = Request.Builder()
+            .headers(headers.toHeaders())
+            .cacheControl(CacheControl.Builder().maxAge(cacheMaxAgeInSeconds, TimeUnit.SECONDS).build())
+            .head()
```

Comment:
@mnonnenmacher @fviernau could one of you please checked whether this `HEAD` request actually works with Artifactory?

Reply:
Yes, it works.